com.unity.multiplayer.mlapi/Prototyping/NetworkNavMeshAgent.cs

```diff
@@ -79,7 +79,7 @@ private void Update()
                 }
             }
 
-            if (NetworkManager.PredictedTime.Time - m_LastCorrectionTime >= CorrectionDelay) // TODO this is not aliased correctly, is this an issue?
+            if (NetworkManager.LocalTime.Time - m_LastCorrectionTime >= CorrectionDelay) // TODO this is not aliased correctly, is this an issue?
```

Comment:
IIRC in general PredictedTime was the preferred time for these computations as before; trying to understand the shift to LocalTime here.  Is it something special w.r.t. NavMesh?

Reply:
I changed this overall to `LocalTime` after talking to people inside the team and Morgane from UX. `LocalTime` seemed to best describe this value because on the server this is not a `PredictedTime` but just the local time.